WordPress/Classes/System/RootViewPresenter+EditorNavigation.swift

```diff
@@ -9,30 +9,12 @@ extension RootViewPresenter {
         return Blog.lastUsedOrFirst(in: context)
     }
 
-    func showPostTab() {
-        showPostTab(completion: nil)
-    }
-
-    func showPostTab(completion afterDismiss: (() -> Void)?) {
-        let context = ContextManager.shared.mainContext
-        // Ignore taps on the post tab and instead show the modal.
-        if Blog.count(in: context) > 0 {
```

Comment:
There used to be a second branch here that I removed. The method that it called returns if there are no sites, so there is no need to repeat this code.